grpc/grpc.h not found in Nix shell

I couldn't get your workflow to work with cabal. I get the following:
```
$ nix-shell release.nix -A grpc-haskell.env
these derivations will be built:
  /nix/store/s0f8xkwhy6v4bbcil2wh42s1zr2dy3hk-grpc-haskell-0.0.0.0.drv
  /nix/store/9kvji52yvy20idxnykb5n9p2p2ansv65-ghc-8.0.2-with-packages.drv
building path(s) ‘/nix/store/gpdjv3cz1550s8q0cskagk59n065rzkb-grpc-haskell-0.0.0.0’
[lots of build info…]
$ cabal configure --with-gcc=clang --enable-tests -v3
[lots of output…]
("/usr/bin/clang",["-fno-stack-protector","/run/user/1000/336465782861021530.c","-o","/run/user/1000/278722862233665123","-E","-D__GLASGOW_HASKELL__=800","-Dlinux_BUILD_OS=1","-Dx86_64_BUILD_ARCH=1","-Dlinux_HOST_OS=1","-Dx86_64_HOST_ARCH=1","-Idist/build/autogen","-Iinclude","-I.","-std=c99","-I/nix/store/6br4gxw0vm4a74p2a0dv24khh8hfpy0k-unix-compat-0.4.3.1/lib/ghc-8.0.2/unix-compat-0.4.3.1/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/process-1.4.3.0/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/directory-1.3.0.0/include","-I/nix/store/6i8n3y5jmh53hql5bqr79r4wxbk7hm2n-vector-0.11.0.0/lib/ghc-8.0.2/vector-0.11.0.0/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/unix-2.7.2.1/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/bytestring-0.10.8.1/include","-I/nix/store/mhf2hjlb2644lw2f2y6iv91kqwnvw07s-primitive-0.6.1.0/lib/ghc-8.0.2/primitive-0.6.1.0/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/time-1.6.0.1/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/base-4.9.1.0/include","-I/nix/store/c82hj05bwbyp20pn433c7x0l8yp3bw3w-gmp-6.1.1-dev/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/integer-gmp-1.0.0.1/include","-I/nix/store/hg0ppf88q1rw0q7x3zqhlxbnviq0ddcg-ghc-8.0.2/lib/ghc-8.0.2/include"])
/usr/bin/clang returned ExitFailure 1 with error message:
In file included from /run/user/1000/336465782861021530.c:1:
./include/grpc_haskell.h:4:10: fatal error: 'grpc/grpc.h' file not found
#include <grpc/grpc.h>
^~~~~~~~~~~~~
1 error generated.
cabal: Missing dependency on a foreign library:
* Missing (or bad) header file: include/grpc_haskell.h
* Missing C library: grpc
This problem can usually be solved by installing the system package that
provides this library (you may need the "-dev" version). If the library is
already installed but in a non-standard location then you can use the flags
--extra-include-dirs= and --extra-lib-dirs= to specify where it is.
If the header file does exist, it may contain errors that are caught by the C
compiler at the preprocessing stage. In this case you can re-run configure
with the verbosity flag -v3 to see the error messages.
```

I checked, `grpc` is in my Nix store and it has the include `grpc/grpc.h`. But Cabal does not seem able to see it.

--- a/README.md
+++ b/README.md
@@ -47,7 +47,7 @@ used to give you a development environment where you can use the `cabal` and
 
 ```bash
 $ nix-shell release-nix -A grpc-haskell.env
-[nix-shell]$ cabal configure --with-gcc=clang --enable-tests && cabal build && cabal test
+[nix-shell]$ cabal configure --enable-tests && cabal build && cabal test
 ```
 
 ```bash

--- a/release.nix
+++ b/release.nix
@@ -3,7 +3,7 @@
 #     $ # Consider adding the following command to your `~/.profile`
 #     $ NIX_PATH="${NIX_PATH}:ssh-config-file=${HOME}/.ssh/config:ssh-auth-sock=${SSH_AUTH_SOCK}"
 #     $ nix-shell -A grpc-haskell.env release.nix
-#     [nix-shell]$ cabal configure --with-gcc=clang --enable-tests && cabal build && cabal test
+#     [nix-shell]$ cabal configure --enable-tests && cabal build && cabal test
 #
 # This will open up a Nix shell where all of your Haskell tools will work like
 # normal, except that all dependencies (including C libraries) are managed by

